### DISCUSSION TITLE
Move markers smoothly when position changed

### DISCUSSION THREAD

**Discussion by AEidSPL** [giorgiabosello/google-maps-react-markers#182] [Help]
Is there a way to move markers smoothly when its position change instead of jump to new position ?

**Comment by giorgiabosello** [comment]
@AEidSPL What do you mean with "_move markers smoothly when its position change_"? When you create and add new ones?

**Reply by AEidSPL** [reply]
Like If position of marker stored in state and this state is updated . I want to figure out how move the to new position with smooth movement 
for example
https://github.com/giorgiabosello/google-maps-react-markers/assets/105779946/8860c87c-000b-4ded-822d-821219a54703

**Reply by giorgiabosello** [reply]
@AEidSPL Mmh, there's no way to do it natively, I think you need to find a way to calculate coordinates from one point to another and update the marker position accordingly.
Or, I don't know, changing its translate prop to move and, once in the final destination, update its coordinates so you won't see the change.
